trillian/db/init.sql

```diff
@@ -0,0 +1,164 @@
+-- ---------------------------------------------
+-- Tree stuff here
+-- ---------------------------------------------
+
+-- Enable strict mode, so invalid data on inserts/updates is treated as error
+-- instead of warning.
+-- https://dev.mysql.com/doc/refman/5.7/en/sql-mode.html#sql-mode-strict
+SET GLOBAL sql_mode = 'STRICT_ALL_TABLES';
+
+CREATE DATABASE trillian;
```

Comment:
How would that help? You mean a container to build a container? At that point I'd rather have the build script do the configuration. The reason why we have containers to build containers is to isolate dependencies, but the only dependencies we have are Go and Docker - I'm not sure if it's worth trying to isolate those into containers themselves.